packages/components/buttons/flat-button/src/flat-button.spec.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { render } from '../../../test-utils';
-import { PlusThinIcon } from '../../icons';
+import { PlusThinIcon } from '@commercetools-uikit/icons';
```

Comment:
Wondering if these (in specs) should be added to `devDependencies`.

Reply:
It's not necessary, the ui-kit packages are available anyway, the other deps can be added to the root package.json as dev deps